### PR TITLE
Clean up sample data

### PR DESCRIPTION
**Pull request by thekaveman**
Following up to #958 - we actually don't need some of the sample data anymore. Also make the sample data we do need more closely align with production data for easier testing.

Part of #776

### CODE REVIEW

**Review by angela-tran — DISMISSED** [review]

**Comment by thekaveman** [comment]
Rebasing changes to fix conflicts...

File: benefits/core/migrations/0002_sample_data.py [hunk 1]
```diff
@@ -95,35 +95,19 @@ def load_sample_data(app, *args, **kwargs):
     EligibilityVerifier = app.get_model("core", "EligibilityVerifier")
 
     verifier1 = EligibilityVerifier.objects.create(
```
Comment (thekaveman) [inline benefits/core/migrations/0002_sample_data.py:97]: c85f4d2c5d930572f70a6b8ac1f19d232fa3b906 is the critical commit of the PR - I removed the _extra_ sample verifier we had (previously standing in as a placeholder for the DMV), and moved the existing sample OAuth verifier up and renamed it to `verifier1`.

This had a big impact on the PO files

**Comment by thekaveman** [comment]
I couldn't get this to run at first... make sure you run `DJANGO_LOAD_SAMPLE_DATA=true bin/init.sh`

**Review by angela-tran — APPROVED** [review]

**Review by machikoyasuda — APPROVED** [review]
image
🙏

**Pull request merged**